packages/legend-application-query/src/components/QueryEditor.tsx

```diff
@@ -190,11 +201,216 @@ const renderQueryEditorHeaderLabel = (
+                value={searchText}
+                placeholder="Search a query by name"
+              />
+              {!searchText ? (
+                <div className="query-editor__query-loader__search-section__input__search__icon">
+                  <SearchIcon />
+                </div>
+              ) : (
+                <>
+                  <button
+                    className="query-editor__query-loader__search-section__input__clear-btn"
+                    tabIndex={-1}
+                    onClick={clearQuerySearching}
+                    title="Clear"
+                  >
+                    <TimesIcon />
+                  </button>
+                </>
+              )}
+            </div>
+          </div>
+          <div className="query-editor__query-loader__body">
+            {editorStore.loadQueriesState.hasCompleted && (
+              <>
+                {editorStore.queries.length > 0 && (
+                  <>
+                    <table className="table">
+                      <thead style={{ position: 'sticky', top: -1 }}>
+                        <tr>
+                          {['Name', 'Author', 'Version'].map((column, idx) => (
+                            <th
+                              key={column}
+                              style={{ fontWeight: 'bold' }}
+                              className="table__cell--left"
+                            >
+                              {column}
+                            </th>
+                          ))}
+                        </tr>
+                      </thead>
+                      <tbody>
+                        {(isMineOnly
+                          ? editorStore.queries.filter(
+                              (q) => q.isCurrentUserQuery,
+                            )
+                          : editorStore.queries
+                        ).map((query) => (
+                          <tr
+                            key={query.id}
+                            className={clsx(
+                              'query-editor__query-loader__body__table__row',
+                              {
+                                'query-editor__query-loader__body__table__row--selected':
+                                  selectedQueryID === query.id,
+                              },
+                            )}
+                            onClick={(event) => setSelectedQueryID(query.id)}
+                          >
+                            <td className="table__cell--left">{query.name}</td>
+                            <td className="table__cell--left">
+                              {query.owner ?? 'anonymous'}
+                            </td>
+                            <td className="table__cell--left">
+                              {query.versionId}
+                            </td>
+                          </tr>
+                        ))}
+                      </tbody>
+                    </table>
```

Comment:
Did you miss my previous comment on this? We should not show this anymore. If you insist, we need to do this slightly differently

Reply:
Are you referring to this one? i think i already deleted it as requested. 
<img width="891" alt="Screen Shot 2022-09-12 at 3 56 14 PM" src="https://user-images.githubusercontent.com/73408381/189744084-a2368d37-c7d8-44b9-b170-a4ea17882f59.png">

Reply:
strangely, I saw it in my review, nvm then, looks like it's gone now!

Reply:
thanks! resolved this comment!